client/src/actions/newGrant.ts

```diff
@@ -66,7 +66,7 @@ export const grantCreated = ({
 });
 
 export const publishGrant =
-  (grantId?: string) =>
+  (grantId?: string, updating?: boolean) =>
```

Comment:
`updating` is redundant, we can figure out if we are updating or creating based on whether `grantId` is undefined or not

Reply:
Thanks for pointing this out